app/views/services/_flow_layout.html.erb

```diff
@@ -20,10 +20,10 @@
   <% else %>
 
     <% if item[:type] == 'flow.branch' %>
-      <article class="flow-item flow-branch" aria-label="<%= item[:title] %>" data-fb-id="<%= item[:uuid] %>">
-        <%= image_pack_tag "diamond.svg",  :usemap => "#thumbnail-#{item[:uuid]}" %>
+      <article class="flow-item flow-branch" data-fb-id="<%= item[:uuid] %>">
+        <%= image_pack_tag "diamond.svg",  usemap: "#thumbnail-#{item[:uuid]}", alt: "Branching diamond", role: "presentation" %>
```

Comment:
The alt text does not provide meaningful content, and therefore benefit, to a user. It is acceptable to leave the alt value empty and, since you've added the role=presentation attribute this should (guess) still provide a way for programmatic analysis of a page to determine the empty alt is valid for accessibility.